thunder/core/prims.py

```diff
@@ -3492,3 +3493,20 @@ def embedding_backward_meta(grad, indices, num_weights, padding_idx, scale_grad_
 
 
 embedding_backward = make_prim(PrimIDs.EMBEDDING_BACKWARD, "embedding_backward", meta=embedding_backward_meta)
+
+
+def input_as_output_meta(
```

Comment:
I'm not totally sure I understand the semantics of this primitive. 

Would it be easier to add a primitive, like `add_`, whose semantics were that it performed the addition inplace on the first operand?

Reply:
I think the motivation is for operator like batch_norm that has inplace update of the inputs(running mean/var), we want an internal prim op to mark it and let nvfuser update this input tensor in-place